source/drivers/payload.cpp

```diff
@@ -3,40 +3,41 @@
 #include "source/updater/updater.h"
 
 err_t handlePayload(uint8_t* buffer, uint8_t buffer_len) {
-    if (buffer[0] == PAYLOAD_FIRMWARE_PART_PACKET) {
-        // payload[0:3] = image_base_address
-        // payload[4:7] = image_patch_address
-        // payload[8] = patch_length
-        // payload[9:9+patch_length-1] = patch, applied sequentially from
-        // image_patch_address
+    uint8_t packet_length = buffer[0];
+    uint16_t packet_checksum = buffer[1] << 8 | buffer[2];
```

Comment:
**Reporter**: CPPCHECK
**Rule**: unreadVariable
**Severity**: INFO
**File**: source/drivers/payload.cpp L7

Variable &#39;packet_checksum&#39; is assigned a value that is never used.. Variable &#39;packet_checksum&#39; is assigned a value that is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a2096771079>*